patterns/newsletter-subscription.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Newsletter Subscription
+ * Slug: twentytwentyfour/newsletter-subscription
+ * Categories: buttons
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"15rem","bottom":"15rem","right":"10rem","left":"10rem"},"blockGap":"2.5rem"}},"backgroundColor":"custom-brick","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-custom-brick-background-color has-background" style="padding-top:15rem;padding-right:10rem;padding-bottom:15rem;padding-left:10rem"><!-- wp:image {"align":"center","id":22,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image aligncenter size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/message_icon.png" alt="" class="wp-image-22"/></figure>
```

Comment:
Please replace `get_stylesheet_directory_uri()` with `get_template_directory_uri()` to load images from assets